readthedocs/proxito/tests/test_urls.py

```diff
@@ -16,8 +16,6 @@ def test_root(self):
         self.assertEqual(match.args, ())
         self.assertEqual(
             match.kwargs, {
-                'subproject_slug': None,
-                'subproject_slash': None,
```

Comment:
This is a good question.

I haven't found anything about this change in the documentation. In theory, nothing should have changed because these functions are alias 🤷🏼 

`subproject_slug` and `subproject_slash` are optional (see https://github.com/readthedocs/readthedocs.org/blob/bebdcbbfe0aaf5599daddcd0f21b42e8df93c8c2/readthedocs/proxito/urls.py#L171-L180) and we have them defined as `None` by default in the view (see https://github.com/readthedocs/readthedocs.org/blob/bebdcbbfe0aaf5599daddcd0f21b42e8df93c8c2/readthedocs/proxito/views/serve.py#L58-L59). So, I assume this won't change any behavior. However, I don't understand why changing from `url` to `re_path` in this version stop matching non-present variables as `None` 😕

Reply:
Note that I haven't change anything else for these URLs: https://github.com/readthedocs/readthedocs.org/pull/8717/files#diff-3377fe6174da17c97cec6956c778d9e987ebd54670d52ac9574c39253a23aecdL170

Reply:
I opened #8809 to track this and try to explain this change in behavior.